config/application.rb

```diff
@@ -78,6 +78,10 @@ class Application < Rails::Application
 
     config.exceptions_app = self.routes
 
+    config.generators do |g|
```

Comment:
We could also use haml-rails instead of doing this, which also provides generators for haml templates.